# Message that holds the current list of mac addresses.
# Address in slot 0 corresponds to robot_0, and so on.
string[] addresses
